src/modules/partners/partner_service.ts

```diff
@@ -5,15 +5,20 @@ export namespace Partner {
   export const findAll = async (requestQuery: Entity.RequestQuery): Promise<Entity.ResponseFindAll> => {
     const items: any = await Repository.findAll({
       name: requestQuery.name,
-      perPage: requestQuery.perPage,
-      currentPage: requestQuery.currentPage
+      per_page: requestQuery.per_page,
+      current_page: requestQuery.current_page
     })
 
     const result: Entity.ResponseFindAll = {
       data: items.data,
       meta: {
-        lastUpdate: items.data.length > 0 ? items.data[0].created_at : null,
-        ...items.pagination
+        last_update: items.data.length ? items.data[0].created_at : null,
+        current_page: items.pagination.currentPage,
+        from: items.pagination.from,
+        last_page: items.pagination.lastPage || 0,
+        per_page: items.pagination.perPage,
+        to: items.pagination.to,
+        total: items.pagination.total || 0,
```

Comment:
penasaran doangnih sama typescript..
i dont know what its return of `items.pagination.total` , if struct always return 0 for number data type.
maybe u don't  need to use it ` || 0`

Reply:
hmm okay, kirain bakal selalu default return 0 sih jika false/null, kalo data typenya number (kaya di go)